src/app/sitting-records/submit-sitting-records/submit-sitting-records.component.html

```diff
@@ -0,0 +1,52 @@
+<main class="govuk-main-wrapper " id="main-content" role="main">
+    <h1 class="govuk-heading-l">Submit Sitting Records</h1>
+
+    <div class="govuk-button-group">
+      <button class="govuk-button govuk-button--secondary" data-module="govuk-button">
+        Submit records
+      </button>
+    </div>
+
+    <div class="govuk-button-group">
+        <a class="govuk-link" (click)="goBack()">Cancel</a>
+    </div>
+
+    <div class="govuk-grid-row">
+      <div [ngClass]="showFilters ? 'govuk-grid-column-three-quarters' : 'govuk-grid-column-full'">
+        <p class="govuk-body govuk-!-margin-bottom-4">
+          Submit Sitting records for {{tribService}}, {{region}}, up to and including the {{date}}
+        </p>
+    
+        <ng-container style="width: 100%;">
+          <table 
+            datatable 
+            [dtOptions]="dtOptions" 
+            [dtTrigger]="dtTrigger"
+            class="govuk-table"
+            id="sittingRecordViewTable">
+            <caption></caption>
```

Comment:
change from using <caption> to using aria-describedby= against table tag with description of table

Reply:
Done